src/UI/WB.UI.Designer/questionnaire/src/views/App/components/VerificationDialog.vue

```diff
@@ -25,16 +25,17 @@
                                     <span class="error-code">[{{ error.code }}]:</span>{{ error.message }}
                                 </div>
                                 <ul class="verification-items" :class="{
-        singleError: !error.isGroupedMessage
-    }">
+                                    singleError: !error.isGroupedMessage
+                                }">
                                     <template v-for="referencesWithErrors in error.errors">
                                         <li class="verification-item-container"
                                             v-for="reference in referencesWithErrors.references"
                                             @click="navigateTo(reference)">
                                             <a class="verification-item" href="javascript:void(0);"
                                                 data-bs-placement="right" data-bs-toggle="tooltip" data-bs-html="true"
                                                 :title="referencesWithErrors.compilationErrorMessages != null ? referencesWithErrors.compilationErrorMessages.slice(0, 10).join('<br />') + (referencesWithErrors.compilationErrorMessages.length > 10 ? '<br />...' : '') : ''"
-                                                data-bs-custom-class="error-tooltip in" data-bs-container='body'>
+                                                data-bs-custom-class="error-tooltip in" data-bs-container='body'
+                                                :data-bs-title="referencesWithErrors.compilationErrorMessages != null ? referencesWithErrors.compilationErrorMessages.slice(0, 10).join('<br />') + (referencesWithErrors.compilationErrorMessages.length > 10 ? '<br />...' : '') : ''">
```

Comment:
two places use the same calculation of title, please extract this code to method